lib/ruby_warrior/abilities/talk.rb

```diff
@@ -0,0 +1,24 @@
+module RubyWarrior
+  module Abilities
+    class Talk < Base
+      def description
+        "Initiates a conversation between the warrior and an informant."
+      end
+
+      def perform(direction = :forward)
+        verify_direction(direction)
```

Comment:
I wonder about this. You technically don't need to face someone to talk to them, even if it can be rude not to.

Reply:
I think this would be important if the warrior has to pick an interaction between multiple characters, eg) attack the sludge on the left and then talk to the informant on the right. This might not be as much of a problem if we keep the towers a single row

Reply:
That is a fair point. The direction will be important if surrounded by multiple informants, which is possible. Let's disregard this comment then.